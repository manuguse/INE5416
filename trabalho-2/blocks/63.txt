1 1 7 7 10 10 10 18 18 21
1 7 7 10 10 13 13 18 18 18
1 8 8 8 8 8 13 13 18 18
2 8 9 9 9 9 14 13 13 19
2 8 9 9 9 14 14 14 19 19
3 3 3 11 11 14 14 15 15 19
4 4 4 11 11 11 11 15 20 19
5 5 4 11 12 15 15 15 20 22
4 4 4 6 12 16 16 17 20 20
6 6 6 6 6 6 17 17 20 20