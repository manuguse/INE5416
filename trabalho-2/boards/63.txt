4 - - - - 2 - 2 - -
- 1 - 3 - 1 6 - - -
1 - 7 - 5 - - 3 - -
- - 3 - - 7 - 2 - -
- 2 1 - - 2 4 - 1 -
2 - - - 4 - - - - 3
7 2 4 - - - 1 - - -
- - - - - - 3 4 - -
6 - - 7 - - - - - 4
1 2 4 - 5 - 3 - - 1