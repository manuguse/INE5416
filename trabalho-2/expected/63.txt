4 2 3 4 5 2 1 2 7 1
3 1 2 3 4 1 6 1 6 4
1 6 7 1 5 3 5 3 5 3
2 4 3 4 6 7 6 2 4 5
1 2 1 2 5 2 4 5 1 4
2 1 3 7 4 1 3 6 2 3
7 2 4 6 3 2 1 5 6 2
2 1 3 5 2 1 3 4 5 1
6 5 1 7 1 2 1 2 3 4
1 2 4 3 5 6 3 1 2 1